docs/UserGuide.md

```diff
@@ -148,6 +142,31 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Uploading cards : `upload`
+
+Imports cards to LingoGO! using a CSV file.
+
+Format: `upload CSV_FILE_PATH`
+
+* The file path can be absolute or relative to the location of the LingoGO! file.
+* The CSV file must have 2 columns. The first column is for English and the second column is for Foreign language.
+  ![sample CSV file](images/SampleCSVFile.png)
+
+Examples:
+* `upload ./dictionary.csv` will load all cards stored in the dictionary.csv to LingoGO!
```

Comment:
`upload ./dictionary.csv` ~~will load all cards stored in the dictionary.csv to LingoGO!~~

Leave examples purely for the commands. All explanations should be in the bullet points before.